fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1449,9 +1449,8 @@ void addTagMapping(GetKeyServerLocationsReply& reply, ProxyCommitData* commitDat
 	for (const auto& [_, shard] : reply.results) {
 		for (auto& ssi : shard) {
 			auto iter = commitData->storageCache.find(ssi.id());
-			if (iter != commitData->storageCache.end()) {
-				reply.resultsTagMapping.emplace_back(ssi.id(), iter->second->tag);
-			}
+			ASSERT_WE_THINK(iter != commitData->storageCache.end());
```

Comment:
I was addressing Jingyu's review comment here. This is not related to the propagation of version vector deltas. Sorry for mixing these in the same commit.